backend/core/socketio_config.py

```diff
@@ -34,16 +163,60 @@ async def sio_disconnect(sid):
 
 @sio_server.on("chats")
 async def chat_message(sid, msg):
-    """Receive a user chat message from FE and put in chat replies queue"""
-    logger.info(f"Server received: sid[{sid}] msg: {msg}")
-    await rabbitmq_client.producer(
-        body=json.dumps(msg),
-        routing_key=RABBITMQ_QUEUE_USER_CHAT_REPLIES,
-    )
+    try:
+        logger.info(f"Server received: sid[{sid}] msg: {msg}")
+        # Receive a user chat message from FE and put in chat replies queue
+        conversation_envelope = msg.get("conversation_envelope", {})
```

Comment:
A missing envelope should result in an error. Is that the case? Here is looks like missing field like `client_phone_number` are just set to `None` and the code proceeds instead of bombing out. Please do not use empty defaults, but throw an error. The faster things break, the easier they are to debug.